driver/src/price_finding/linear_optimization_price_finder.rs

```diff
@@ -233,9 +234,12 @@ fn write_input(input_file: &str, input: &str) -> std::io::Result<()> {
 }
 
 fn run_solver(input_file: &str) -> Result<(), PriceFindingError> {
+    let optimization_config: String =
```

Comment:
I'd prefer if this can be passed in from above (as a solver config option or the like). Reading the env var directly in this component makes it hard to unit test (env vars are considered global state).